docs/manuals/core/concepts/sequence.md

```diff
@@ -0,0 +1,32 @@
+A `Sequence^` is designed to model an algorithm. It represents a direct acyclic graph of input, intermediate, and output
+data nodes linked together by tasks. A _sequence_ is a set of tasks designed to perform a set of functions.
+
+For instance, in a typical machine learning application, we may have several sequences: a _sequence_ dedicated to
```

Comment:
I don't think you need to use italics on the 'sequence' word any longer: we now know what that is.
(same below)